examples/aks-cluster/kustomization.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generatorOptions:
+  disableNameSuffixHash: true 
+
+configMapGenerator:
+
+- namespace: crossplane-system
+  name: pythonic-aks
+  options:
+    labels:
+      function-pythonic.package: 'aks'
```

Lets put your modules in the root path, which allows the pipeline steps to work unmodified.
```suggestion
      function-pythonic.package: ''
```